src/components/swap/components/trade-details/trade-info.tsx

```diff
@@ -26,14 +33,15 @@ const TradeInfoItemRow = (props: { item: TradeInfoItem }) => {
             {title}
           </Text>
         </Flex>
-        {isLink === true && (
+        {isLoading && <StyledSkeleton width={60} />}
+        {!isLoading && isLink === true && (
           <Link isExternal href={values[1]}>
             <Text fontSize='12px' fontWeight='700' textColor={colors.icGray3}>
               {shortenAddress(values[0])}
             </Text>
           </Link>
         )}
-        {isLink === undefined && (
+        {!isLoading && isLink === undefined && (
```

Comment:
Can `isLink === false` be a valid case/need to be handled too?

Reply:
Good point, will check.